lib/bootstrap/bundler.rb

```diff
@@ -103,10 +108,10 @@ def invoke!(options = {})
       # force Rubygems sources to our Gemfile sources
       ::Gem.sources = ::Gem::SourceList.from(options[:rubygems_source]) if options[:rubygems_source]
 
-      ::Bundler.settings[:path] = LogStash::Environment::BUNDLE_DIR
-      ::Bundler.settings[:gemfile] = LogStash::Environment::GEMFILE_PATH
-      ::Bundler.settings[:without] = options[:without].join(":")
-      ::Bundler.settings[:force] = options[:force]
+      ::Bundler.settings.set_local(:path, LogStash::Environment::BUNDLE_DIR)
+      ::Bundler.settings.set_local(:gemfile, LogStash::Environment::GEMFILE_PATH)
+      ::Bundler.settings.set_local(:without, options[:without])
```

Comment:
The change to pass through the `options[:without]` array directly without colon-joining its elements into a string isn't called out in this PR. Is this change intentional?

Reply:
Thankfully we don't need to worry about this! the `set_local` method calls `set_key`, [which converts arrays into colon-joined strings](https://github.com/bundler/bundler/blob/v1.17.1/lib/bundler/settings.rb#L352-L360)

Reply:
Ah! Thanks for clarifying.